### PR TITLE
Firefox iOS 9.0 String Export (2017-08-21)

### PR DESCRIPTION
**Pull request by st3fan**
Please don't merge yet. Reviewing.

### CODE REVIEW

**Comment by st3fan** [comment]
I think this is good to go.

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -1521,6 +1584,30 @@
...
+      <trans-unit id="Settings.DNT.OptionAlwaysOn">
+        <source>Always</source>
+        <note>DNT Settings option for always on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.OptionOnWithTP">
+        <source>Only when using Tracking Protection</source>
+        <note>DNT Settings option for only turning on when Tracking Protection is also on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.Title">
+        <source>Send websites a “Do Not Track” signal that you don’t want to be tracked</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: I see there's still the same apostrophe nit that Flod raised in https://github.com/mozilla-l10n/firefoxios-l10n/pull/33: "You're mixing up proper apostrophes and straight quotes in strings. This string uses ’, for example intro.Slides.Private.Title uses '."

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -926,6 +917,78 @@
...
+      <trans-unit id="FxAPush_DeviceDisconnected_ThisDevice_title">
+        <source>Sync Disconnected</source>
+        <note>Title of a notification displayed when this device has been disconnected by another device.</note>
+      </trans-unit>
+      <trans-unit id="FxAPush_DeviceDisconnected_UnknownDevice_body">
+        <source>A device has disconnected from Firefox Sync</source>
+        <note>Body of a notification displayed when unnamed device has been disconnected from FxA.</note>
+      </trans-unit>
+      <trans-unit id="FxAPush_DeviceDisconnected_body">
+        <source>Firefox Sync has disconnected %@</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: Unless I'm confused, shouldn't this have changed to  "%S has been successfully disconnected.", as per ryanfeeley's comment in https://github.com/mozilla-l10n/firefoxios-l10n/pull/33 ?

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -1597,6 +1684,14 @@
         <source>New Tab Settings</source>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.Status">
+        <source>When Opening Firefox</source>
+        <note>Status text of setting to enable the Go to Copied URL feature - https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff]: Flod mentioned to add a note here: "E.g.: Description displayed under the ”Offer to open copied link” option."

**Comment by flodolo** [comment]
Confirmed, it looks like changes from the previous round didn't make it into the PR.

**Comment by st3fan** [comment]
All issues have been addressed in #36 so I am closing this PR.

**Pull request closed without merging**